app/views/admin/categories/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="row">
+    <div class="col-md-6 centered">
+        <div class="jumbotron">
+            <div>
+                <h2><%= @category.title %></h2>            
+            </div>
+            <p><%= @category.description %></p>
+        </div>
+
+        <div>
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th class="center">Words</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <% @category.words.each do |word| %>
+                        <tr>
+                            <td>
+                                <%= link_to word.content, admin_category_word_url(word, category_id: @category.id) %>
```

Comment:
@CharlotteDS Use path helpers in view pages

Reply:
You can improve this here
![image](https://user-images.githubusercontent.com/20507565/49853969-c22f7280-fe23-11e8-9d1c-b706bdc7392f.png)